mlir/test/Dialect/LLVMIR/inlining.mlir

```diff
@@ -56,3 +56,122 @@ func.func @test_not_inline(%ptr : !llvm.ptr) -> () {
   call @with_mem_attr(%ptr) : (!llvm.ptr) -> ()
   return
 }
+
+// -----
+
+// Include all function attributes that don't prevent inlining
+llvm.func internal @callee() -> (i32) attributes { function_entry_count = 42 : i64, dso_local } {
```

Comment:
Maybe also add an alternative calling convention here.

Reply:
Do you by any chance know how to do this? It seems to implicitly be set to "C", if I set it as a named attribute I get a duplicate error.

Reply:
`CConv` is a default valued attribute that has a specified position in the assembly format. Thus you cannot pass it in the attribute directory, as the parser will resolve the default value when it's not present in the predefined location. 
To properly pass this in, use something like the following:

```
llvm.func internal fastcc @cconv() {
  llvm.return
}
```

Reply:
Thanks!